src/UserSpaceInstrumentation/AllocateInTracee.h

```diff
@@ -14,9 +14,11 @@
 namespace orbit_user_space_instrumentation {
 
 // Allocate `size` bytes of memory in the tracee's address space using mmap. The memory will have
-// read, write, and execute permissions.
+// read, write, and execute permissions. The memory allocated will start at `address`. If `address`
```

Comment:
not exactly. since you are not using MAP_FIXED the address is a hint, kernel will try to allocate it at the specified address if possible and if not it will allocate nearby that address for some value of nearby. If the expected behaviour to allocate at exact address there should be additional check of return value of mmap, and some handling if this did not happen.

Reply:
Great, thanks, I did not know about MAP_FIXED which does exactly what we need. I planned to handle possible misplacement "above" this code (in which case the comment would have been misleading). 
With MAP_FIXED I can adjust the code to the comment.

Reply:
well, there is another thing you should probably know before doing that - MAP_FIXED will create the map even if there was already another map at this address. It will "override" it, and I am not sure this is safe.